### PR TITLE
LA-255 Clear Search View After Copying

### PR DESCRIPTION
**Pull request by imGok**
#255 

clearsearch

### CODE REVIEW

**Comment by imGok** [comment]
Basically, "TextEditingController.clear()" is needed to clear the search bar and it was a stateless widget so now we can clear the search term outside the widget

**Comment by hoangdat** [comment]
Can you update this PR?

**Comment by hoangdat** [comment]
make it work with `features_v2`

**Comment by hoangdat** [comment]
@huynguyennovem please review this PR also

File: lib/presentation/di/module/widget_module.dart [hunk 1]
```diff
@@ -446,4 +449,10 @@ class WidgetModule {
         ));
   }
 
+  void _provideHomeAppBarComponent() {
+    getIt.registerFactory(() => HomeAppBarWidget());
```
Comment (huynguyennovem) [inline lib/presentation/di/module/widget_module.dart:453]: Registered `HomeAppBarWidget` but there is no usage of it with `getIt`

Comment (imGok) [reply lib/presentation/di/module/widget_module.dart:453]: you are right I just need the viewmodel

**Review by hoangdat — APPROVED** [review]

**Pull request merged**